### PR TITLE
Diawang/pure k8s log restserver

### PR DESCRIPTION
**Pull request by wangdian**

### CODE REVIEW

File: src/kube-runtime/src/runtime [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (abuccts) [inline src/kube-runtime/src/runtime]: what if there's no `bash` in image?

File: src/kube-runtime/src/runtime.d/sshd.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (abuccts) [inline src/kube-runtime/src/runtime.d/sshd.sh]: same

File: src/kube-runtime/src/runtime.d/user.sh [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (abuccts) [inline src/kube-runtime/src/runtime.d/user.sh]: same

File: src/kube-runtime/build/kube-runtime.dockerfile [hunk 4]
```diff
@@ -20,11 +20,11 @@ FROM python:2.7-alpine3.8
 
 ARG BARRIER_DIR=/opt/frameworkcontroller/frameworkbarrier
 
-WORKDIR /usr/local/pai
+RUN mkdir -p /kube-runtime/src
```
Comment (abuccts) [inline src/kube-runtime/build/kube-runtime.dockerfile]: do not need, duplicate with `WORKDIR`

File: src/kube-runtime/build/kube-runtime.dockerfile [hunk 5]
```diff
@@ -20,11 +20,11 @@ FROM python:2.7-alpine3.8
...
+WORKDIR /kube-runtime/src
 
 COPY src/ ./
 COPY --from=frameworkcontroller/frameworkbarrier:v0.3.0 $BARRIER_DIR/frameworkbarrier ./init.d
-RUN mkdir -p ./logs && \
-    chmod -R +x ./
+RUN chmod -R +x ./
 
-CMD ["/bin/sh", "-c", "/usr/local/pai/init"]
+CMD ["/bin/sh", "-c", "/kube-runtime/src/init"]
```
Comment (abuccts) [inline src/kube-runtime/build/kube-runtime.dockerfile]: You can move the sources in dockerfile

```suggestion
CMD ["/bin/sh", "-c", "mv ./* /usr/local/pai && /usr/local/pai/init"]
```

File: src/kube-runtime/src/runtime (hunk 1 above)
Comment (wangdian) [reply src/kube-runtime/src/runtime]: Ubuntu is our officially supported image, bash should be installed. Using /bin/sh may call dash shell and cause unexpected result.

**Pull request merged**